examples/src/main/scala/App.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+import com.microsoft.hyperspace._
+import com.microsoft.hyperspace.Hyperspace
+import com.microsoft.hyperspace.index.{IndexConfig, IndexConstants}
+
+object App {
+  def main(args: Array[String]): Unit = {
+    // Create Spark session
+    val sparkConf = new SparkConf().setMaster("local[*]")
```

Comment:
Does it need to be local?  What if I try to run this app on a cluster?

Reply:
I had added Local to make the code runnable right after checkout. However, I looked at how Spark does it for its examples and followed the same pattern. As an example, check [here](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala#L39).